lib/standard/string_nit.c

```diff
@@ -11,10 +11,12 @@
 
 #include "string_nit.h"
 
+// Returns the length of `recv` as a `char*` (excluding the null character)
+long native_int_length_str(long recv){
+	return snprintf(NULL, 0, "%ld", recv);
```

Comment:
Have you considered doing this in Nit? It would probably require using floats, but it would be interesting to compare the perf... or look at how snprintf is implemented.

~~~
var length = (self.pow10 + 1).floor
if self < 0 then length += 1
~~~